scripts/update_robots.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+#------------------------------------------------------------------------------
+# Prints version of the Solidity compiler that the source code corresponds to.
+#
+# The documentation for solidity is hosted at:
+#
+#     https://docs.soliditylang.org
+#
+# ------------------------------------------------------------------------------
+# This file is part of solidity.
+#
+# solidity is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# solidity is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with solidity.  If not, see <http://www.gnu.org/licenses/>
+#
+# (c) 2017 solidity contributors.
+#------------------------------------------------------------------------------
+
+set -e
+set -u
+
+PATH_TO_GET_VERSION_SCRIPT="./scripts/get_version.sh"
+PATH_TO_ROBOTS_TXT_FILE="docs/_static/robots.txt"
+TEMP_FILE="temp_store_robots.txt"
+VERSION_FROM_SCRIPT=`bash ${PATH_TO_GET_VERSION_SCRIPT} | perl -pe 'chomp'`
```

Comment:
You don't need to explicitly run `bash` to execute a script.

Also, I don't think the variable is necessary. It's more readable to use script name directly.

Finally, I think the name should say what this version refers to, not where it comes from.

```suggestion
solidity_version=`./scripts/get_version.sh | perl -pe 'chomp'`
```